README.md

```diff
@@ -23,6 +23,15 @@ You can read more about the various `.env` files that you can use at:
 
 - https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env
 
+**Note:** Before you run the next steps, you will need to ensure you have access to the Firebase project you want to use. This access can be set up through the Firebase web UI. The following variables are required to be set up:
+
+```bash
+REACT_APP_PROJECT_ID=
+REACT_APP_API_KEY=
+REACT_APP_APP_ID=
+REACT_APP_MEASUREMENT_ID=
+```
+
```

Comment:
This seems like it might be better closer to/as a part of/directly after this line:

> In the main `sparkle` folder, copy `.env.example` to a new file `.env.local`, and fill it with the appropriate details.

As they are both relating to setting up the ENV vars.